### PR TITLE
feat(redis): allowed ips MTA-2004c

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**
- Manage allowed IPs
- ajouté au menu

### CODE REVIEW

**Review by bene2k1 — APPROVED** [review]

File: databases/databases-for-redis/how-to/create-a-database-for-redis.mdx [hunk 1]
```diff
@@ -64,5 +64,5 @@ Database for Redis™ is a low-latency caching solution based on in-memory data
 6. Click **Create Database for Redis™** to confirm your choices and launch creation.
```
Comment (nzerhouni) [inline databases/databases-for-redis/how-to/create-a-database-for-redis.mdx:64]:
```suggestion
6. Click **Create a Redis™ Database Instance** to confirm your choices and launch creation.
```

File: databases/databases-for-redis/how-to/manage-allowed-ip-addresses-redis.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- You have a [Database for Redis™](/databases/databases-for-redis/how-to/create-a-database-for-redis)
+
+</Message>
+
+## How to add an IP address
+
+1. Click **Databases Redis™** under **Database** on the side menu. A list of your Database Instances displays.
```
Comment (nzerhouni) [inline databases/databases-for-redis/how-to/manage-allowed-ip-addresses-redis.mdx:24]:
```suggestion
1. Click **Databases Redis™** under **Databases** on the side menu. A list of your Database Instances displays.
```

File: databases/databases-for-redis/how-to/manage-allowed-ip-addresses-redis.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  validation: 2022-07-12
+  posted: 2019-07-12
+---
+
+Allowed IPs define remote access restrictions for your Database Instances. The default entry `0.0.0.0/0` enables any host to establish a connection.
+
+<Message type="requirement">
+
+- You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+- You have a [Database for Redis™](/databases/databases-for-redis/how-to/create-a-database-for-redis)
```
Comment (nzerhouni) [inline databases/databases-for-redis/how-to/manage-allowed-ip-addresses-redis.mdx:18]:
```suggestion
- You have a [Redis™ Database Instance](/databases/databases-for-redis/how-to/create-a-database-for-redis)
```

File: databases/databases-for-redis/how-to/manage-allowed-ip-addresses-redis.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  </Message>
+
+## How to delete an IP address
+
+1. Go to your allowed IP address list.
+2. Click <Icon name="delete" />.
+3. Click **Delete your IP**.
+
+<Navigation title="See Also">
+  <PreviousButton to="/databases/databases-for-redis/how-to/create-a-database-for-redis">How to create a Database for Redis™</PreviousButton>
```
Comment (nzerhouni) [inline databases/databases-for-redis/how-to/manage-allowed-ip-addresses-redis.mdx:49]:
```suggestion
  <PreviousButton to="/databases/databases-for-redis/how-to/create-a-database-for-redis">How to create a Redis™ Database Instance</PreviousButton>
```

File: databases/databases-for-redis/how-to/scale-up-a-database-for-redis.mdx [hunk 5]
```diff
@@ -65,6 +65,6 @@ If you want to scale up the size of your Database for Redis™, you can edit you
   You can check your new number of nodes in the Database Instance information page under **Node configuration**.
 
 <Navigation title="See Also">
-  <PreviousButton to="/databases/databases-for-redis/how-to/create-a-database-for-redis">How to create a Database for Redis™</PreviousButton>
+  <PreviousButton to="/databases/databases-for-redis/how-to/manage-allowed-ip-addresses-redis">How to manage allowed IPs</PreviousButton>
   <NextButton to="/databases/databases-for-redis/how-to/delete-a-database-for-redis">How to delete a Database for Redis™</NextButton>
```
Comment (nzerhouni) [inline databases/databases-for-redis/how-to/scale-up-a-database-for-redis.mdx:69]:
```suggestion
  <NextButton to="/databases/databases-for-redis/how-to/delete-a-database-for-redis">How to delete a Redis™ Database Instance</NextButton>
```

**Review by nzerhouni — CHANGES_REQUESTED** [review]

**Comment by ldecarvalho-doc** [comment]
errata

**Pull request closed without merging**